### PR TITLE
first_time_ctrl converted to es6

### PR DESCRIPTION
**Pull request by karinashenaz**

### CODE REVIEW

**Comment by Sjors** [comment]
@karinashenaz you still need to delete the original coffee file.

**Comment by karinashenaz** [comment]
@Sjors  deleted

File: assets/js/controllers/first_time_ctrl.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+walletApp.controller("FirstTimeCtrl", ($scope, $modalInstance, firstTime) => {
+  $scope.firstTime = firstTime;
+  return $scope.ok = () => $modalInstance.close(firstTime);
```
Comment (Sjors) [inline assets/js/controllers/first_time_ctrl.js]: Unnecessary return statement.

**Comment by Sjors** [comment]
Still there, did you push the new version?

**Comment by karinashenaz** [comment]
git reset and pushed…
again..
?

**Pull request merged**